modules/custom/movie_reservations/movie_reservations.info.yml

```diff
@@ -0,0 +1,6 @@
+name: Movie Reservations Module
+description: Creates a page that lists available movies.
+package: Custom
+type: module
+#version: 1.0
```

Comment:
Da li su nam potrebni komentari? takodje gde vidis ovaj crveni krug sa minusom unutra, to znaci da nemas prazan red na kraju fajla, pa dodaj to gde fali.